src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyDiscoveryManager.cs

```diff
@@ -77,12 +79,6 @@ public ProxyDiscoveryManager(ITestRequestSender testRequestSender, ITestRuntimeP
         /// </summary>
         public void Initialize()
```

Comment:
Removing this will spiral into cleaning Discovery Interface, & even execution interface, I suggest that this be taken as part of some clean up routine.?